SpatialGDK/Source/SpatialGDK/Private/Interop/ActorSystem.cpp

```diff
@@ -2390,8 +2394,15 @@ bool ActorSystem::HasPendingOpsForChannel(const USpatialActorChannel& ActorChann
 	return bHasPendingCreateEntityRequests;
 }
 
+bool ActorSystem::IsCreateEntityRequestInFlight(Worker_EntityId EntityId) const
+{
+	return CreateEntityRequestsInFlight.Contains(EntityId);
+}
+
 void ActorSystem::OnEntityCreated(const Worker_CreateEntityResponseOp& Op, FSpatialGDKSpanId CreateOpSpan)
 {
+	CreateEntityRequestsInFlight.Remove(Op.entity_id);
```

Comment:
To my other comment - this should be on authority gain. This might come on the same tick or might not.